crates/chain/src/indexed_tx_graph.rs

```diff
@@ -14,7 +14,7 @@ use crate::{
 /// A struct that combines [`TxGraph`] and an [`Indexer`] implementation.
 ///
 /// This structure ensures that [`TxGraph`] and [`Indexer`] are updated atomically.
-#[derive(Debug)]
+#[derive(Clone, Debug)]
```

Comment:
Why is this change necessary? Ideally we want to avoid cloning `IndexedTxGraph` at it may become problematic.

Reply:
This was probably left over from old code/debugging use and has been removed.